frappe/tests/test_commands.py

```diff
@@ -226,16 +226,16 @@ def test_execute(self):
 
 		# test 2: execute a command expecting an errored output as local won't exist
```

Comment:
Do update this comment as well, ideally add another test so both situations are covered?

Reply:
https://github.com/frappe/frappe/pull/27753

Reply:
https://github.com/frappe/frappe/pull/27763

Reply:
Ha! Thanks, this _did_ catch a missing exit status, fixed.